upup/models/config/components/kube-apiserver/_do/kube-apiserver.do.options

```diff
@@ -0,0 +1,2 @@
+KubeAPIServer:
+  CloudProvider: external
```

Comment:
Noted. It was really difficult trying to figure out who to change the CloudProvider flag for kube-apiserver, I'll follow up with this in another PR